runelite-client/src/main/java/net/runelite/client/plugins/cluescrolls/clues/EmoteClue.java

```diff
@@ -103,6 +103,15 @@ public class EmoteClue extends ClueScroll implements LocationClueScroll
 			.orElseGet(Stream::empty)
 			.map(ItemRequirements::item)
 			.toArray(SingleItemRequirement[]::new));
+	private static final AnyRequirementCollection ANY_RUNE_HERALDIC_SHIELD = any("Rune shield (h)",
+		ItemVariationMapping.getVariations(RUNE_SHIELD_H1).stream()
+			.map(ItemRequirements::item)
+			.toArray(SingleItemRequirement[]::new));
+	private static final AnyRequirementCollection ANY_RUNE_HERALDIC_HELM = any("Rune helm (h)",
+		ItemVariationMapping.getVariations(RUNE_FULL_HELM).stream()
```

Comment:
Assuming the below comment re: item variation mapping is addressed separately, the following change should also happen.
```suggestion
		ItemVariationMapping.getVariations(RUNE_HELM_H1).stream()
```